App templates: Add the `Hide the Menu in the Closed State` article (#1029)
DevExpress/devextreme-documentation: concepts/40 Angular Components/10 Getting Started/02 Create a DevExtreme Application/30 Configure the Navigation Menu/20 Hide the Menu in the Closed State.md

> In the closed state, the navigation menu is still visible displaying item icons. If the items lack icons, it is better to entirely hide the menu. To do this, open the `SideNavOuterToolbarComponent` or `SideNavInnerToolbarComponent` (depending on the used [layout](/Documentation/Guide/Angular_Components/Getting_Started/Create_a_DevExtreme_Application/#Layouts)) and change the `updateDrawer()` function as follows:

Reviewer: >menu is still visible displaying item icons

This sentence is a bit complicated to me. Maybe smth like 'The navigation menu is not entirely hidden in the closed state because it displays item icons'?

Author: @AlexJameson 
I was about to apply your suggestion but noticed that the word "entirely" would be repeated. How about this version?

```suggestion
In the closed state, the navigation menu is still partially visible because it displays item icons. If the items lack icons, it is better to entirely hide the menu. To do this, open the `SideNavOuterToolbarComponent` or `SideNavInnerToolbarComponent` (depending on the used [layout](/Documentation/Guide/Angular_Components/Getting_Started/Create_a_DevExtreme_Application/#Layouts)) and change the `updateDrawer()` function as follows:
```

> In the closed state, the navigation menu is still partially visible because it displays item icons. If the items lack icons, it is better to entirely hide the menu. To do this, open the `SideNavOuterToolbarComponent` or `SideNavInnerToolbarComponent` (depending on the used [layout](/Documentation/Guide/Angular_Components/Getting_Started/Create_a_DevExtreme_Application/#Layouts)) and change the `updateDrawer()` function as follows:

Reviewer 2: "still partially" Adverb adverb?

Reviewer 2: If something is visible, it is visible?

Reviewer 2: "it is better to" Why is it better to do this? This sounds more like an opinion than a fact.

Reviewer 2: ```suggestion
In the closed state, the navigation menu is visible because it displays item icons. If the items do not have icons, you can hide the menu. To do this, open the `SideNavOuterToolbarComponent` or `SideNavInnerToolbarComponent` (depending on the [layout](/Documentation/Guide/Angular_Components/Getting_Started/Create_a_DevExtreme_Application/#Layouts)) and change the `updateDrawer()` function as follows:
```
